NodeManager.ino

```diff
@@ -35,6 +35,54 @@ void before() {
   */
   
 
+   nodeManager.setRebootPin(A0);
```

Comment:
Absolutely, i work with guys that are involved in web dev, so this approach make them truly happy ;)
And as i use lots of nodes, in 2 minutes you can switch to code for one or another and that's really a breath!